python-libraries/data-platform-catalogue/data_platform_catalogue/client/base.py

```diff
@@ -8,7 +8,7 @@
     DataProductMetadata,
     TableMetadata,
 )
-from ..search_types import ResultType, SearchResponse
+from ..search_types import MultiSelectFilter, ResultType, SearchResponse
```

Comment:
⛏️ I appreciate it's an absolute (pardon the pun) minefield, but do we want to standardise relative vs. absolute imports within the package?

Here it's relative, [here it's absolute](https://github.com/ministryofjustice/data-platform/pull/3075/files#diff-f425ca2ec2ca477a29960cc7cde6b0d8aab55fa17eaae0d62de6ce17c0e21fefR11).